doc/task-hierarchy.md

```diff
@@ -28,6 +29,10 @@
     - `Classification -> Verification -> Grammar Verification`: Verify whether a text is grammatical 
     - `Classification -> Verification -> Relevance Verification`
     - `Classification -> Verification -> Answer Verification`: Verify whether a text answers the question
+    - `Classification -> Verification -> Statement Verification`: Verify whether a text statement is true or false
+    - `Classification -> Verification -> Paraphrase Verification`: Verify whether a text sequence is a paraphrase of another text sequence
+    - `Classification -> Verification -> Translation Verification`: Verify whether a text sequence is a translation of another text sequence
```

Comment:
I think we don't need Paraphrase Verification and Translation Verification. Since we have paraphrasing and translation categories, and their combination with verification has the same meaning. I'm not sure if we need Statement Verification either.

Reply:
Thanks a lot for looking into the PR! 

Here's my reasoning behind creating those extra taxonomy branches: 
- "Paraphrasing" is a generation task
- "Classification -> Verification" is a classification task
- "Classification -> Verification -> Paraphrase Verification" is a classification task (inheriting from "Verification" in the taxonomy), similar to its sibling nodes in the taxonomy (relevance/answer verification).

Replacing "Classification -> Verification -> Paraphrase Verification" by the combination ["Paraphrasing", "Classification -> Verification"] looks ambiguous to me: is this a classification or generation task? not sure how it can have the same meaning as "Classification -> Verification -> Paraphrase Verification". 

Any thoughts? thanks!

Reply:
I don't think "Paraphrasing" is a generation task, since it's not under text generation category. It probably can contain any task which is related to paraphrasing.
@ghlai9665 @swarooprm What do you think?